resources/views/volumes/videos/index/meta.blade.php

```diff
@@ -0,0 +1,51 @@
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <h3 class="panel-title">Video information</h3>
+    </div>
+    <div class="panel-body table-responsive">
```

Comment:
No need to wrap this into a `.panel-body`, this will only add an unnecessary padding around the table (see images/index/meta.blade.php).